packages/nc-gui/layouts/shared-view.vue

```diff
@@ -1,7 +1,26 @@
 <script lang="ts" setup>
+import { useRouter } from 'vue-router'
 import { navigateTo } from '#app'
 const { isLoading, currentVersion } = useGlobal()
 const { sharedView } = useSharedView()
+const router = useRouter()
+
+if (window.parent !== window) {
```

Comment:
Add a comment what this code block is supposed to do, otherwise nobody else will understand what this was supposed to do 😄